pyproject.toml

```diff
@@ -201,5 +201,4 @@ fixable = ["A", "B", "C", "D", "E", "F", "R", "S", "W", "U"]
 unfixable = []
 
 [project.scripts]
-run_topostats = "topostats.run_topostats:main"
-toposum = "topostats.plotting:main"
+topostats = "topostats.entry_point:entry_point"
```

Comment:
I think for now we should leave the existing entry points in place and have this as an additional entry point that we can develop and add to. This avoids having to switch whole-sale but allows testing by users who are inclined to do so and allows us to add in the modular processing as we go through refactoring.